marketplace_builder/views/pages/get-started/partner-portal/setting-up-production-domain.liquid

```diff
@@ -0,0 +1,82 @@
+---
+converter: markdown
+metadata:
+  title: Setting Up a Production Domain
+  description: This guide will help you set up a production domain. 
+  slug: get-started/partner-portal/setting-up-production-domain
+searchable: true
+---
+
+This guide will help you set up a production domain. The production domain is the domain of your production Instance, your live site available to visitors. 
+
+## Requirements
+To be able to set up a production domain on the Partner Portal, you have to have access to our Partner Portal, have a production Instance, and you should be familiar with the recommended development workflow. 
+
+* [Accessing Our Partner Portal](/get-started/setup/accessing-partner-portal)
+* [Setting Up a Site on Our Platform](https://documentation.platform-os.com/get-started/setup/setting-up-site)
+* [Development Workflow](https://documentation.platform-os.com/get-started/setup/development-workflow)
+
+## Steps
+
+Setting up a production domain is a five-step process:
+
+1. Add new domain
+2. Add hosted zone with alias record
+3. Request SSL Certificate
+4. Create CNAME record (for DNS validation)
```

Comment:
4th point is an optional scenario in case user uses DNS validation for SSL Certificate

Reply:
@godot Thank you! Edited.